### PR TITLE
Reverse a Doubly Link List Problem Added

### PR DESCRIPTION
**Pull request by SamarpanCoder2002**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

New Problem(Reverse a Doubly Link List.cpp) Added through C-plus-plus->ds->Reverse_a_Doubly_Link_List.cpp
(This pull request is associated with https://github.com/TesseractCoding/NeoAlgo/issues/2457#issue-826501841)
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Excited to contribute here..... This problem is new here..... Hope You will merge that pull request...Thank you..

### CODE REVIEW

**Comment by SamarpanCoder2002** [comment]
> ### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
> yes
> 
> ### Description
> New Problem(Reverse a Doubly Link List.cpp) Added through C-plus-plus->ds->Reverse_a_Doubly_Link_List.cpp
> (This pull request is associated with [#2457 (comment)](https://github.com/TesseractCoding/NeoAlgo/issues/2457#issue-826501841))
> 
> ### Checklist
> * [x]  I've read the contribution guidelines.
> * [x]  I've checked the issue list before deciding what to submit.
> * [x]  I've edited the `README.md` and link to my code.
> 
> ## Related Issues or Pull Requests
> Excited to contribute here..... This problem is new here..... Hope You will merge that pull request...Thank you..

Please Merge This Pull Request.... If any problem here to merge this, you can feel free to tell me...Thanks

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -84,6 +84,11 @@
 - [Binary Search Tree to Greater Sum Tree](ds/BST_to_Greater_Sum_Tree.cpp)
 - [Threaded Binary Tree](ds/ThreadedBinaryTree.cpp)
 
+- [Reverse a Doubly Link List](ds/Reverse_a_Doubly_Link_List.cpp)
+
+- [Stack using queue](ds/stack/Stack_using_queue.cpp)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md:89]: Why are there two Markdown updates?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 
- Add a beginner-friendly description about your Code at the top
- Add description of functions 
- Add sample I/O at the end of the Code
- Add a new line at the EOF
- Add complexity at the end of the code
- Use a Code formatter to format the Code

**Comment by atarax665** [comment]
Closing due to inactivity.

**Pull request closed without merging**